charters/wot-ig-2019.html

```diff
@@ -368,7 +368,7 @@ <h3 id="external-coordination">External Organizations</h3>
             <dt><a href="http://www.iiconsortium.org">Industrial Internet Consortium</a></dt>
             <dd>For collaboration on a testbed and gathering user requirements for the Web of Things.</dd>
             <dt><a href="https://opcfoundation.org">OPC Foundation</a></dt>
-            <dd>For collaboration on semantic interoperability and end to end security across platforms.</dd>
+            <dd>For collaboration on semantic interoperability in terms of information model and protocol binding.</dd>
```

Comment:
I agree that adding the information model and protocol binding makes sense, however I would not remove security.
Suggest to call it:
"For collaboration on semantic interoperability in terms of information model, protocol binding and security.

Reply:
OPC UA uses standard security approaches. So far, I do not see any investigation into this. However, I can keep it since it doesn't hurt.